101-traffic-manager-virtual-machines/azuredeploy.json

```diff
@@ -0,0 +1,271 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
```

Comment:
Thinking about it more, I've decided to make the web app and vm templates into 201 series.  While the customer scenario is less complex them template itself is more difficult to understand and grasp.  Keeping the external endpoint as a 101 because there is nothing other than the traffic manager profile resource.